src/contracts/Airdrop.h

```diff
@@ -0,0 +1,226 @@
+using namespace QPI;
+
+#define AIRDROP_START_SUCCESS 0
+#define AIRDROP_INSUFFICIENT_FUND 1
+#define AIRDROP_STARTED 2
+#define AIRDROP_SEND_FUND 3
+#define AIRDROP_START_ERROR 4
+#define AIRDROP_INCORRECT_TIME 5
+#define AIRDROP_DISTRIBUTE_SUCCESS 6
+#define AIRDROP_DISTRIBUTE_FAILD 7
+#define AIRDROP_PERIOD_LIMITED 8
+#define AIRDROP_FUND_LIMITED 9
+#define AIRDROP_INSUFFICIENT_TOKEN 10
+
+struct AirdropLogger
+{
```

Comment:
Logger structs should always have the following at the beginning of the struct:
`uint32 contractId; // to distinguish bw SCs (set when LOG_*() is called`
`uint32 type; // Assign a random unique (per contract) number to distinguish messages of different types (not mandatory at the moment)`